tests/test_hintfonts.py

```diff
@@ -21,9 +22,13 @@ def __init__(self, inpath, outpath):
         self.verbose = False
 
 
+def _temp(tmpdir, path):
+    return str(tmpdir / basename(path))
```

Comment:
this seems like an odd way of assembling a path. I'm surprised that it even works on Windows. I wonder if tox is fixing it under the hood.

Reply:
I don’t have the slightest idea how it works, I just reshuffled @anthrotype’s code.

Reply:
pytest `tmpdir` fixture is a `py.path.LocalPath` object, here is the api:
http://py.readthedocs.io/en/latest/path.html
https://docs.pytest.org/en/latest/tmpdir.html

Reply:
the `py` module is part of pytest (originally, pytest was part of it, hence the old name py.test)